example/integration_test/360_test.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:integration_test/integration_test.dart';
+import 'package:polar/polar.dart';
+
+import '../../test/tests.dart';
+
+const identifier = 'E0729E20';
+
+void main() async {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  await requestPermissions();
+  testSearch(identifier);
+  testConnection(identifier);
+
+  /// TODO: Add FTU
+  testBasicData(identifier);
+  testBleSdkFeatures(
+    identifier,
+    features: PolarSdkFeature.values.toSet().difference({
+      PolarSdkFeature.offlineRecording,
+      PolarSdkFeature.h10ExerciseRecording,
+    }),
+  );
+  testStreaming(
+    identifier,
+    features: {
```

Comment:
This should test all available streaming features. Is this everything the 360 supports? Does the 360 not support location?

Reply:
SDK mode might not have all of them. What does `getAvailableOnlineStreamDataTypes` return?